internal/printer/service_test.go

```diff
@@ -204,7 +204,7 @@ func Test_ServiceConfiguration(t *testing.T) {
 
 	stopPortForwardingButtonGroup := component.NewButtonGroup()
 	stopPortForwardingButtonGroup.Config = component.ButtonGroupConfig{Buttons: []component.Button{
-		component.NewButton("Stop port forward", action.Payload{
+		*component.NewButton("Stop port forward", action.Payload{
```

Comment:
Should not have to use `*`

Reply:
Not sure about this one 'cause we are creating and array of objects, so you need to pass the references of the objects right?

Reply:
So, I was checking about this and found that I can change

```
type ButtonGroupConfig struct {
	// Buttons are buttons in the group.
	Buttons []Button `json:"buttons"`
}
```

To be

```
type ButtonGroupConfig struct {
	// Buttons are buttons in the group.
	Buttons []*Button `json:"buttons"`
}
```

But, with that change, it means that the line you commented would need this change:

```
stopPortForwardingButtonGroup.Config = component.ButtonGroupConfig{Buttons: []*component.Button{
              component.NewButton("Stop port forward", action.Payload{

```

Because now the array is going to be of references.

So before making the change, just want to make sure that is what we want.

Reply:
all right this is fine for now, we can leave this alone